### ISSUE TITLE
Form Validation - Focus indicator on an input fields is not 3:1

### ISSUE THREAD

**Issue by hydegeoff** [wet-boew/wet-boew#9967]
The colour of focus indicator on an input field is only 2.4:1 colour contrast (at its darkest point) when on the WET form validation examples page.  It is especially problematic when it is a drop-down, no text cursor that is blinking to identify where the focus is on the page.   See the screenshot below 

Tested using:  

- Testing agents: Chrome version 135.0.7049.115 (Official Build) (64-bit)  
- Colour Contrast Analyser (CCA) Version 3.1.4 
- WET Form Validation page - Date modified: 2025-02-26 

According to WCAG 2.1 SC-2.4.7, the focus indicator should be at least 3:1.  I believe the focus indicator needs to be a darker to ensure it meets this success criteria.  

![Image](https://github.com/user-attachments/assets/8ad2752e-2446-4d91-b4a3-839628d4daae)
